recipes/fretbursts/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "FRETBursts" %}
+{% set version = "0.5.6" %}
+{% set sha256 = "368b5448b2c6e6a35c6a2842867eb3c962a6eedf02778995872630dd517cf77c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win32 and py35]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
+    - cython
+    - numpy
```

Comment:
Because you use both cython and numpy, I think that you might be using numpy's C API.  If you are, then you need to pin your versions here to x.x.  The reason is that you need to have the same version at runtime that was used at build time.

    - numpy   x.x